source/common/access_log/access_log_formatter.cc

```diff
@@ -350,6 +371,26 @@ StreamInfoFormatter::StreamInfoFormatter(const std::string& field_name) {
         return UnspecifiedValueString;
       }
     };
+  } else if (field_name == "DOWNSTREAM_PEER_URI_SAN") {
+    field_extractor_ =
+        sslConnectionInfoStringExtractor([](const Ssl::ConnectionInfo& connection_info) {
+          return connection_info.uriSanPeerCertificate();
```

Comment:
When I look at how ` SslSocket::uriSanPeerCertificate()` implemented today, it seems that it should really be returning all SAN. Should we fix this, so that we create template patterns that are stable and forward compatible? CC @PiotrSikora

Reply:
Sure I don't mind fixing that. I'll just have all current usage just use the first SAN from the new function